### ISSUE TITLE
Clangd Can't Find STD Library Headers After Installing Boost

### ISSUE THREAD

**Issue by jcmfisker** [clangd/clangd#1174]
Please describe the problem.
I'm additionally using the VSCode clangd plugin with Remote Explorer.  I use CMake to with `CMAKE_EXPORT_COMPILE_COMMANDS` flag set to 1.  Code completion, highlighting, signatures, ... all work as expected before installing Boost.  After installing boost, clangd does not parse standard header files.  It always reports `"<std header>" not found`.

If you can, provide a minimal chunk of code that shows the problem (either inline, or attach it if larger).

sudo apt install libboost-all-dev

**Logs**
Log when trying to #include std::array header
`
I[19:12:00.350] <-- textDocument/definition(112)
V[19:12:00.350] ASTWorker running Definitions on version 1557 of <path_to_application_code>
...
V[19:12:00.350] No heuristic index definition for array
I[19:12:00.350] --> reply:textDocument/definition(112) 0 ms
V[19:12:00.350] >>> {
  "id": 112,
  "jsonrpc": "2.0",
  "result": []
}
`
**System information**

Output of `clangd --version`:

clangd version 14.0.3 (https://github.com/llvm/llvm-project 1f9140064dfbfb0bbda8e51306ea51080b2f7aac)
Features: linux+grpc
Platform: x86_64-unknown-linux-gnu

Editor/LSP plugin:

VSCode
clangd plugin

Operating system:
Description:    Ubuntu 20.04.3 LTS
Release:        20.04
Codename:       focal

**Comment by HighCommander4** [comment]
Sounds very strange, but there isn't a lot of info to go on here.

Could you post a complete log of a clangd editing session that involves opening a file that shows an unresolved include diagnostic?

**Comment by kadircet** [comment]
hi @jcmfisker as @HighCommander4 pointed out it's really hard to follow up on such issues in the absence of logs (sometimes even with logs). feel free to re-open once you can attach some logs.
